Simplenote/Simplenote-Info.plist

```diff
@@ -34,6 +34,14 @@
 			<key>URL Identifier</key>
 			<string>Simplenote URL</string>
 		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>simplenotemac</string>
```

Comment:
Since Electron also has the `simplenote://` scheme, adding `simplenotemac://` to avoid collisions